module/os/linux/zfs/zpl_file.c

```diff
@@ -161,17 +161,60 @@ static int
 zpl_fsync(struct file *filp, loff_t start, loff_t end, int datasync)
 {
 	struct inode *inode = filp->f_mapping->host;
+	znode_t *zp = ITOZ(inode);
+	zfsvfs_t *zfsvfs = ITOZSB(inode);
 	cred_t *cr = CRED();
 	int error;
 	fstrans_cookie_t cookie;
 
+	ZPL_ENTER(zfsvfs);
+	ZPL_VERIFY_ZP(zp);
+	/*
+	 * the variables z_sync_writes_cnt and z_async_writes_cnt work in
```

Comment:
```suggestion
	 * The variables z_sync_writes_cnt and z_async_writes_cnt work in
```

Proper capitalization throughout this comment and the next.

Reply:
fixed, i've also added capitalization to the comments in zfs_vnops_os.c
& in mmap_sync_001_pos.ksh i've updated the script's description to make the purpose of the test clearer.